Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackFormConfiguration.swift

```diff
@@ -0,0 +1,161 @@
+     */
+    @objc public var addScreenshotButtonLabel: String = "Add a screenshot"
+    
+    /**
+     * The label of the button to remove the screenshot from the form.
+     * - note: Default: `"Remove screenshot"`
+     * - note: ignored if `enableScreenshot` is `false`.
+     */
+    @objc public var removeScreenshotButtonLabel: String = "Remove screenshot"
+    
+    // MARK: Name
+    
+    /**
+     * Requires the name field on the feedback form to be filled in.
+     * - note: Default: `false`
+     */
+    @objc public var isNameRequired: Bool = false
+    
+    /**
+     * Displays the name field on the feedback form.
+     * - note: Default: `true`
+     * - note: ignored if `isNameRequired` is `true`.
+     */
+    @objc public var showName: Bool = true
+    
+    /**
+     * The label of the name input field.
+     * - note: Default: `"Name"`
+     * - note: ignored if `showName` is `false`.
+     */
+    @objc public var nameLabel: String = "Name"
+    
+    /**
+     * The placeholder for the name input field.
+     * - note: Default: `"Your Name"`
+     * - note: ignored if `showName` is `false`.
+     */
+    @objc public var namePlaceholder: String = "Your Name"
+    
+    // MARK: Email
+    
+    /**
+     * Requires the email field on the feedback form to be filled in.
+     * - note: Default: `false`
+     */
+    @objc public var isEmailRequired: Bool = false
+    
+    /**
+     * Displays the email field on the feedback form.
+     * - note: Default: `true`
+     * - note: ignored if `isEmailRequired` is `true`.
+     */
+    @objc public var showEmail: Bool = true
+    
+    /**
+     * The label of the email input field.
+     * - note: Default: `"Email"`
+     */
+    @objc public var emailLabel: String = "Email"
+    
+    /**
+     * The placeholder for the email input field.
+     * - note: Default: `"your.email@example.org"`
+     */
+    @objc public var emailPlaceholder: String = "your.email@example.org"
+    
+    // MARK: Buttons
+    
+    /**
+     * The label of the submit button used in the feedback form.
+     * - note: Default: `"Send Bug Report"`
+     */
+    @objc public var submitButtonLabel: String = "Send Bug Report"
+    
+    /**
+     * The label of cancel buttons used in the feedback form.
+     * - note: Default: `"Cancel"`
+     */
+    @objc public var cancelButtonLabel: String = "Cancel"
+    
+    /**
+     * The label of confirm buttons used in the feedback form.
+     * - note: Default: `"Confirm"`
+     */
+    @objc public var confirmButtonLabel: String = "Confirm"
+    
+    // MARK: Theme
+    
+    /**
+     * Builder for light mode theme overrides.
+     * - note: Default: `nil`
+     */
+    @objc public var lightThemeOverrides: ((SentryUserFeedbackThemeConfiguration) -> Void)?
+    
+    /**
+     * Builder for dark mode theme overrides.
+     * - note: Default: `nil`
+     */
+    @objc public var darkThemeOverrides: ((SentryUserFeedbackThemeConfiguration) -> Void)?
```

Comment:
Ah you're right, we do simply need `theme`/`themeOverrides`, at least for iOS versions older than those that support dark mode, and it can default to light mode. 

For those iOS versions that do support dark mode, we should check which mode the OS is currently in, and use the light or dark config.

Reply:
Hmm, actually, what do you think about just having two: `themeOverrides` and `darkThemeOverrides`?

Reply:
Yeah, Im fine with `themeOverrides` and `darkThemeOverrides`